index.html

```diff
@@ -58,8 +58,8 @@ <h4 class="jumbotron-text">Project Jupyter exists to develop open-source softwar
                         <img class="img-responsive" src="assets/labpreview.png" id="portal" alt="examples of jupyterlab workspaces in single document and multiple document workspaces"/>
                     </div>
                     <div class="col-md-6 nb-highlight-text">
-                        <h3>JupyterLab: Jupyter’s Next-Generation Notebook Interface</h3>
-                        <h4 class="nb-desc">JupyterLab is a web-based interactive development environment for Jupyter notebooks, code, and data. JupyterLab is flexible: configure and arrange the user interface to support a wide range of workflows in data science, scientific computing, and machine learning. JupyterLab is extensible and modular: write plugins that add new components and integrate with existing ones.</h4>
+                        <h3>JupyterLab: The Next-Generation Notebook</h3>
```

Comment:
Thakns again for working on the wording. I believe that the word _interface_ should be kept JupyterLab is not a new _notebook_. The notebook format is the same between JupyterLab and other interfaces.

Reply:
Roger. I've put that word back in.